spec/models/item_spec.rb

```diff
@@ -48,67 +48,70 @@
-      ItemOrder.create(item: paper, price: paper.price, quantity: 3, order_id: order_1.id)
-      ItemOrder.create(item: pencil, price: pencil.price, quantity: 1, order_id: order_1.id)
-
-      expect(Item.popularity('desc')).to eq([album, journal, ahgabong, cardboard, dolls])
-    end
-
-    it ".popularity for least" do
-      shop1 = Merchant.create(name: "Mike's Print Shop", address: '123 Paper Rd.', city: 'Denver', state: 'CO', zip: 80203)
-      shop2 = Merchant.create(name: "K-Pop Black Market", address: '123 Bike Rd.', city: 'Denver', state: 'CO', zip: 80203)
-
-      #Shop 2 items
-      cardboard = shop2.items.create(name: "EXO Kai' Cardboard", description: "Just a cutout, not the real deal!", price: 10000, image: "https://images-na.jpg", inventory: 20)
-      album = shop2.items.create(name: "Super Junior Time Slip", description: "Comeback CD", price: 55, image: "https://images-na.jpg", inventory: 120)
-      ahgabong = shop2.items.create(name: "GOT7 Ahgabong", description: "True Ahgases Only", price: 100, image: "https://images-na.jpg", inventory: 70)
-      dolls = shop2.items.create(name: "All Day6 Member Dolls", description: "Time of your life", price: 600, image: "https://images-na.jpg", inventory: 12)
-
-      #Shop 1 items
-      paper = shop1.items.create(name: "Lined Paper", price: 20)
-      pencil = shop1.items.create(name: "Yellow Pencil", description: "You can write with it!", price: 2, image: "https://images-na.jpg", inventory: 100)
-      journal = shop1.items.create(name: "Bullet Journal", description: "DIY Planner", price: 2, image: "https://images-na.jpg", inventory: 200)
-      fan = shop1.items.create(name: "Personal Fan", description: "Cool Down Whenever", price: 35, image: "https://images-na.jpg", inventory: 100)
-
-      user = User.create(name: "Natasha Romanoff", address: "890 Fifth Avenue", city: "Manhattan", state: "New York", zip: "10010", email: "spiderqueen@hotmail.com", password: "arrow", role: 0)
-      order_1 = Order.create(name: 'Hyram', address: '123 Stang Ave', city: 'Hershey', state: 'PA', zip: 17033, user_id: user.id)
-
-      ItemOrder.create(item: cardboard, price: cardboard.price, quantity: 20, order_id: order_1.id)
-      ItemOrder.create(item: ahgabong, price: ahgabong.price, quantity: 55, order_id: order_1.id)
-      ItemOrder.create(item: album, price: album.price, quantity: 80, order_id: order_1.id)
-      ItemOrder.create(item: journal, price: journal.price, quantity: 70, order_id: order_1.id)
-      ItemOrder.create(item: dolls, price: dolls.price, quantity: 10, order_id: order_1.id)
-      ItemOrder.create(item: fan, price: fan.price, quantity: 2, order_id: order_1.id)
-      ItemOrder.create(item: paper, price: paper.price, quantity: 3, order_id: order_1.id)
-      ItemOrder.create(item: pencil, price: pencil.price, quantity: 1, order_id: order_1.id)
-
-      expect(Item.popularity('asc')).to eq([pencil, fan, dolls, cardboard, ahgabong])
+    
+    describe "class methods" do
```

Comment:
moved the test under class methods, instead of instance methods
(this test doesn't relate with US 42 and 43